src/renderer/state.ts

```diff
@@ -818,10 +848,36 @@ export class AppState {
       return;
     }
 
+    if (this.versionLock) {
+      console.log(`Releasing lock on version ${this.version}`);
+
+      // release the lock on the previous version
+      this.versionLockController.abort();
+
+      // replace the spent AbortController
+      this.versionLockController = new AbortController();
+    }
+
     const { version } = ver;
     console.log(`State: Switching to Electron ${version}`);
     this.version = version;
 
+    navigator.locks.request(
```

Comment:
```suggestion
    await navigator.locks.request(
```

Reply:
That would hang the execution until the lock is released, resulting in a blank screen on startup. For our purposes, we don't actually need the lock to be released to keep going — we just want it to be there.